### DISCUSSION TITLE
How to modify Learning rate in specific Epoch during Training?

### DISCUSSION THREAD

**Discussion by hoya-cho** [open-mmlab/mmdetection#9717] [General]
I know learning rate policy in mmdetection increases linearly past the warm_up iter for the Lr specified in the optimizer up to the set step. After that, it is a structure that decreases from the set step.
The learning rate control provided now is
1. Decrease Lr every few epochs
2. It is Lr reduction in a set step (ex [27:33]).

but how to modify Learning rate in specific Epoch during Training?
